pydash-front/src/authenticated_app/dashboard/statistics_page/StatisticsPage.js.orig

```diff
@@ -0,0 +1,309 @@
+import VisitorsHeatmapPanel from './VisitorsHeatmapPanel';
+import UniqueVisitorsHeatmapPanel from './UniqueVisitorsHeatmapPanel';
+
+// Visual:
+import { withStyles } from 'material-ui/styles';
+import SwipeableViews from 'react-swipeable-views';
+import Tabs, { Tab } from 'material-ui/Tabs';
+import Typography from 'material-ui/Typography';
+import DeleteIcon from 'material-ui-icons/Delete'
+import CreateIcon from 'material-ui-icons/Create'
+import { Button } from 'material-ui';
+import Dialog, { DialogActions, DialogContent, DialogContentText, DialogTitle,} from 'material-ui/Dialog';
+import TextField from 'material-ui/TextField';
+
+function TabContainer({ children, dir }) {
+  return (
+    <Typography component="div" dir={dir} style={{ padding: 8 * 3 }}>
+      {children}
+    </Typography>
+  );
+}
+
+TabContainer.propTypes = {
+  children: PropTypes.node.isRequired,
+  dir: PropTypes.string.isRequired,
+};
+
+const styles = theme => ({
+  root: {
+    backgroundColor: theme.palette.background.paper,
+    width: 500,
+  },
+  settings:{
+    float: "centre",
+  },
+});
+
+
+class StatisticsPage extends Component {
+    constructor(props) {
+        super(props);
+        this.divRef = React.createRef();
+        this.state = {
+            dashboard: null,
+            dashboardName: this.props.dashboard.id,
+            dashboardUrl: this.props.dashboard.url,
+            dashboardToken: '',
+            visits_per_day: [],
+            unique_visitors_per_day: [],
+            average_execution_times: [],
+            error: "",
+            width: 0,
+            open: false,
+            current_tab: 0,
+        };
+    }
+
+    componentDidMount() {
+        console.log(this.divRef);
+        
+        this.setState(prevState => {
+            /* const width =  this.divRef.current.clientWidth;*/
+            const width = window.screen.width;
+            return {...prevState, width: width}
+        })
+
+        axios(window.api_path + '/api/dashboards/' + this.props.dashboard.id, {
+            method: 'get',
+            withCredentials: true
+        }).then((response) => {
+            let total_visits = (response.data && response.data.aggregates && response.data.aggregates.total_visits) || 0;
+            console.log("Response: ", response);
+            if (response.data.hasOwnProperty('error')) {
+                this.setState(prevState => {
+                    return {
+                        ...prevState,
+
+                        dashboard: response.data,
+<<<<<<< HEAD
+                        
+
+=======
+>>>>>>> development
+                        total_visits: total_visits,
+                        error: response.data.error,
+                    }
+                });
+            } else {
+                this.setState(prevState => {
+                    return {
+                        ...prevState,
+
+
+                        dashboard: response.data,
+<<<<<<< HEAD

```

Comment:
still some artefacts of a conflict here

Reply:
wait, what are these `.orig` files for?

Reply:
They are an artefact that Git creates when doing conflict resolution on the command line. I've removed these files.